locations/spiders/gls-deu.py

```diff
@@ -0,0 +1,66 @@
+import scrapy
+import json
+import csv
+
+class GenspiderSpider(scrapy.Spider):
+    name = 'glsdeu'
+    allowed_domains = ['api.gls-pakete.de']
+    index = 1
+    start_urls = ['https://api.gls-pakete.de/parcelshops?version=4&coordinates=50.88246,8.91596&distance=50']
+    usedLongLat = []
+    deList = []
+
+    with open('../searchable_points/eu_centroids_40km_radius_country.csv', 'r') as openFile:
+      results = csv.reader(openFile)
+      for result in results:
+          if(result[3] == 'DE'):
+              data = {
+                  "longitude": result[2],
+                  "latitude": result[1]
+              }
+              deList.append(data)
+
+
+    def parse(self, response):
+
+        firstResults = json.loads(response.body)
+        results = firstResults['shops']
+        for result in results:
+            address = result['address']
+            coordinates = address['coordinates']
+            longitude = coordinates['longitude']
+            latitude = coordinates['latitude']
+            name = address['name']
+            state = 1
+
+            if(GenspiderSpider.usedLongLat):
```

Comment:
I don't understand the need for `usedLongLat`. Each set of coords should only be called once anyway,